lib/options.js

```diff
@@ -108,6 +108,12 @@ module.exports = optionator({
         description: "Report errors only"
     },
     {
+        option: "max-warnings",
+        type: "Number",
+        default: "-1",
+        description: "Show an error if too many warnings"
```

Comment:
Not sure error exit code means anything, I prefer to be as transparent as possible.

Keep playing with it to see if you can get it to not wrap.